clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -30,7 +30,11 @@ class HttpElasticsearchDAO private[dataaccess] (
     with StrictLogging {
 
   private[dataaccess] val httpClient = {
-    val restClient = RestClient.builder(httpHosts: _*).build()
+    val restClient = RestClient
+      .builder(httpHosts: _*)
+      // The default timeout is 100̈ms, which is too slow for query operations.
+      .setRequestConfigCallback(_.setConnectionRequestTimeout(10000))
```

Comment:
Instead of having a hard-coded value here, could we add a new key to [this config blob](https://github.com/broadinstitute/clio/blob/develop/clio-server/src/main/resources/reference.conf#L36) to parameterize the timeout?

Reply:
yeah, good idea